src/index.html

```diff
@@ -3,11 +3,508 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Title</title>
+    <title>Kickstarter</title>
+    <link
+      rel="icon"
+      href="images/favicon.ico"
+      type="image/svg+xml"
+    >
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Barlow:wght@100;300;400;500;700;900&family=IBM+Plex+Sans:wght@100;200;300;400;500;700&family=Inter:wght@100;400;500;700;900&family=Montserrat:wght@100;300;400;500;700;900&family=Playfair+Display&family=Poppins:wght@100;200;300;400;500;600;700;800;900&family=Raleway:wght@400;500;700&family=Roboto:wght@100;300;400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Hello Mate Academy</h1>
+  <body class="page">
+
+    <nav class="nav">
+      <div class="container">
+        <div class="grid-container ">
+          <div class="nav__navigation">
+            <a href="#" class="nav__logo animation">
+              <img src="images/icon/vector.svg" alt="logo">
+            </a>
+
+            <div class="nav__navbar">
+              <a href="#" class="nav__specs animation">SPECS</a>
+
+              <div class="nav__languages">
+                <a
+                  href="#"
+                  class="nav__eng animation active"
+                  data-language="eng"
+                >
+                  EN/
+                </a>
+                <a
+                  href="#"
+                  class="nav__ua animation"
+                  data-language="ua"
+                >
+                  UA
+                </a>
+
+              </div>
+
+              <button class="nav__btn animation">Buy</button>
+              <a
+                class="nav__button animation"
+                id="menu"
+                href="#menu"
+                data-modal=" ">
+              </a>
+            </div>
+
+          </div>
+        </div>
+      </div>
+    </nav>
+
+    <aside class="modal">
+      <div class="container">
+        <div class="grid-container">
+          <div class="modal__navigation">
+            <a href="#" class="modal__logo animation">
+              <img src="images/icon/vector.svg" alt="logo">
+            </a>
+
+            <a data-close=" " class="modal__button animation"></a>
+          </div>
+
+          <div class="modal__navbar">
+            <a href="#benefits" class="modal__link navigation__link modal__bottom">OUR STORY</a>
+            <a href="#about" class="modal__link navigation__link modal__bottom">ABOUT US</a>
+            <a href="#features" class="modal__link navigation__link modal__bottom">FEATURES</a>
```

Comment:
According to BEM methodology navigation__link is redundant here you can pass the styles of this class to modal__link class.